include/claws/array_ops.hpp

```diff
@@ -169,6 +169,24 @@ namespace claws
     SCALAR_ARRAY_OPERATOR_DEF(<<);
     SCALAR_ARRAY_OPERATOR_DEF(>>);
   }
-
   /// @}
+
+  namespace impl
+  {
+    template<size_t size, class Func, size_t... indices>
+    constexpr auto init_array(Func &&functor, std::index_sequence<indices...>)
+    {
+      return std::array{(static_cast<void>(indices), functor())...};
+    }
+  };
```

Comment:
I'm being nitpicky, but the `;` here is not necessary. (I'll edit it from here and then approve the PR)